# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../xen-tools/compile.patch
# Copyright (C) 2006 The T2 SDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- tools/xenstat/xentop/Makefile	2006-04-07 14:24:59.000000000 +0200
+++ tools/xenstat/xentop/Makefile	2006-04-09 16:37:55.634761500 +0200
@@ -26,8 +26,8 @@
 sbindir=$(prefix)/sbin
 
 CFLAGS += -DGCC_PRINTF -Wall -Werror -I$(XEN_LIBXENSTAT)
-LDFLAGS += -L$(XEN_LIBXENSTAT)
-LDLIBS += -lxenstat -lncurses
+LDFLAGS += -L$(XEN_LIBXENSTAT) -L$(XEN_XENSTORE)
+LDLIBS += -lxenstat -lncurses -lxenstore
 
 all: xentop
 
